### PR TITLE
Refactoring docs

### LINKED ISSUE #2: Document available features on R and Python libraries here
All features already implemented should have a good description here along with code examples in each language

### PR DESCRIPTION
**Pull request by fccoelho**
I have started a reorganization of the Documentation to serve as an example for @eduardocorrearaujo and @dcpcamara to follow.

This is related to issue #2

### CODE REVIEW

**Review by eduardocorrearaujo — APPROVED** [review]

File: index.rst [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+.. EpigraphHub_lib documentation master file, created by sphinx-quickstart on Mon Mar 21 11:22:53 2022. You can adapt this file completely to your liking, but it should at least contain the root `toctree` directive.
 
 Welcome to EpigraphHub libraries documentation!
-===========================================
+================================================
+
+The EpiGraphHub library is designed to provide external users with all of the analytical functionality used to power
+the EpiGraphHub platform and applications. In fact, all applications designed to run on the EpiGraphHub platform rely heavily on it.
```
Comment (dcpcamara) [inline index.rst]:
```suggestion
the EpiGraphHub platform and its applications. In fact, all applications designed to run on the EpiGraphHub platform rely heavily on it.
```

File: getting_started.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+To install them, run the following commands:
+
+**Python:**
+```bash
+pip install epigraphhub
+```
+
+**R :**
+```
+install.packages("epigraphhub")
```
Comment (dcpcamara) [inline getting_started.md]:
```suggestion
install.packages("devtools")
devtools::install_github("thegraphnetwork/r-epigraphhub")
```

File: index.rst [hunk 3]
```diff
@@ -1,18 +1,27 @@
...
+
+This library should be used by users that want to contribute to the platform as well as by users that want the borrow
```
Comment (dcpcamara) [inline index.rst]:
```suggestion
This library should be used by users that want to contribute to the platform as well as by users that want to borrow
```

**Review by dcpcamara — CHANGES_REQUESTED** [review]

**Comment by fccoelho** [comment]
Thanks, @dcpcamara, I have accepted all your corrections!

**Review by dcpcamara — APPROVED** [review]

**Pull request merged**